### ISSUE TITLE
The version of python and tensorflow

### ISSUE THREAD

**Issue by hjyai94** [lelechen63/MRI-tumor-segmentation-Brats#4]
Hi, Can you  tell me which version of your python and tensorflow?

**Comment by lelechen63** [comment]
I think the version is not a factor which leads to a large decrease in accuracy.

**Comment by WrathofBhuvan11** [comment]
Bro 12GB GPU !!!? Do you have pre-trained model please send it's google driver link?
